autoscheduler/frontend/src/redux/reducers/courseCards.ts

```diff
@@ -43,12 +50,65 @@ const initialCourseCardArray: CourseCardArray = {
     remote: SectionFilter.NO_PREFERENCE,
     honors: SectionFilter.EXCLUDE,
     asynchronous: SectionFilter.NO_PREFERENCE,
+    sortType: SortType.DEFAULT,
     sections: [],
     loading: true,
     collapsed: false,
   },
 };
 
+/**
+ * Function that sorts sections
+ * @param sections: the sections to be sorted
+ * @param sortType: how the sections should be sorted
+ */
+function sortSections(sections: SectionSelected[], sortType: SortType): SectionSelected[] {
+  /** DEFAULT:
+   *    Groups sections by professor and honors status,
+   *    then sorts each group by the lowest section
+   *    number in the group, with TBA sections getting sorted to the bottom.
+   */
+  if (sortType === undefined || sortType === SortType.DEFAULT) {
```

Comment:
When would `sortType` be undefined?

Reply:
Primarily just in other tests, if you wanted I could probably go through and thouroughly test so this can be removed?